lib/recurly/pager.php

```diff
@@ -8,23 +8,24 @@
  */
 abstract class Recurly_Pager extends Recurly_Base implements Iterator
 {
-  private $_etag;
```

Comment:
I wonder why this was never used. Is it something we want to utilize in the future?